.github/workflows/flutter_test.yml

```diff
@@ -13,7 +13,7 @@ on:
       - "flutter/**"
 
 env:
-  SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+  SENTRY_AUTH_TOKEN_E2E: ${{ secrets.SENTRY_AUTH_TOKEN_E2E }}
 
 jobs:
```

Comment:
FYI it's now preferrable (faster) to run android emulator on linux: https://github.com/ReactiveCircus/android-emulator-runner#running-hardware-accelerated-emulators-on-linux-runners

Reply:
oh yeah, will change it

Reply:
oh yeah, will change it